src/main/java/com/iexec/core/chain/BlockchainConnectionHealthIndicator.java

```diff
@@ -145,19 +161,24 @@ private void connectionFailed() {
     /**
      * Reset {@link BlockchainConnectionHealthIndicator#consecutiveFailures} to {@code 0}.
      * <p>
-     * If never been OUT-OF-SERVICE, then:
+     * If previous failures occurred, then:
      * <ul>
-     *     <li>Reset the {@link BlockchainConnectionHealthIndicator#firstFailure} var to {@code null};</li>
-     *     <li>Log a "connection restored" message.</li>
-     * </ul>
+     * <li>Log a "connection restored" message.
+     * <li>Reset the {@link BlockchainConnectionHealthIndicator#firstFailure} var to {@code null}
+     * <li>If OUT-OF-SERVICE, publish a {@link ChainConnectedEvent} event and reset the OUT-OF-SERVICE state
```

Comment:
It seems the closing `</ul>` is recommended there, based on [the JavaDoc documentation](https://www.oracle.com/fr/technical-resources/articles/java/javadoc-tool.html).
```suggestion
     * <li>If OUT-OF-SERVICE, publish a {@link ChainConnectedEvent} event and reset the OUT-OF-SERVICE state
     * </ul>
```

Reply:
see 4439335